src/shipping/strategies/amazonv2/amazon-payv2-shipping-strategy.ts

```diff
@@ -0,0 +1,82 @@
+import { createAction } from '@bigcommerce/data-store';
+
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { MissingDataError, MissingDataErrorType } from '../../../common/error/errors';
+import { PaymentMethod, PaymentMethodActionCreator } from '../../../payment';
+import { AmazonPayv2PaymentProcessor } from '../../../payment/strategies/amazon-payv2';
+import { ShippingInitializeOptions } from '../../shipping-request-options';
+import { ShippingStrategyActionType } from '../../shipping-strategy-actions';
+import ShippingStrategy from '../shipping-strategy';
+
+export default class AmazonPayv2ShippingStrategy implements ShippingStrategy {
+    private _paymentMethod?: PaymentMethod;
+
+    constructor(
+        private _store: CheckoutStore,
+        private _paymentMethodActionCreator: PaymentMethodActionCreator,
+        private _amazonPayv2PaymentProcessor: AmazonPayv2PaymentProcessor
+    ) {}
+
+    updateAddress(): Promise<InternalCheckoutSelectors> {
+        return Promise.resolve(this._store.getState());
+    }
+
+    selectOption(): Promise<InternalCheckoutSelectors> {
+        return Promise.resolve(this._store.getState());
```

Comment:
I guess shipping address will be provided by Amazon so `updateAddress` can be a no-op method. But shipping options are not the same. I don't know how a shopper can select a shipping option if it's a no-op.